README.md

```diff
@@ -2,8 +2,14 @@
 
 This Swift package provides support for automatically printing crash backtraces of Swift programs.
 
+The library is designed to fill a gap in backtraces support for Swift on non-Darwin platforms. 
+When this gap is closed at the language runtime level, this library will become redundant and be deprecated.
+
 ## Usage
 
+In most cases, direct usage of this library is discouraged. 
```

Comment:
Why do we discourage it? Swift Lifecycle is great but I have written many tools that simply don't need a lifecycle to speak of. They'll still use `swift-backtrace` but I wouldn't want to add `swift-lifecycle` there.

Reply:
agreed, the library is useful on it own in a class of utilities and tools that do not need a shutdown hook or dont have a complex shutdown procedure

the idea to add such comment came up in the last SSWG meeting, but perhaps that opinion was biased towards web-services and daemons that mostly do need lifecycle

maybe a better way to phrase this is:

```
When building web-services and daemons, direct usage of this library is discouraged. 
 
Instead, use [swift-service-lifecycle](https://github.com/swift-server/swift-service-lifecycle) which helps manage the application lifecycle including setting up backtraces hooks when needed. 
```

wdyt?

Reply:
I like it , thanks!